comp/core/workloadmeta/types.go

```diff
@@ -782,19 +897,38 @@ func (d KubernetesDeployment) String(verbose bool) string {
 
 var _ Entity = &KubernetesDeployment{}
 
+// ECSTaskKnownStatusStopped is the known status of an ECS task that has stopped.
+const ECSTaskKnownStatusStopped = "STOPPED"
+
+// ECSTaskTags is a map of tags for an ECS task.
+type ECSTaskTags map[string]string
+
+// ContainerInstanceTags is a map of tags for an ECS container instance.
+type ContainerInstanceTags map[string]string
+
 // ECSTask is an Entity representing an ECS Task.
 type ECSTask struct {
 	EntityID
 	EntityMeta
-	Tags                  map[string]string
-	ContainerInstanceTags map[string]string
-	ClusterName           string
-	Region                string
-	AvailabilityZone      string
-	Family                string
-	Version               string
-	LaunchType            ECSLaunchType
-	Containers            []OrchestratorContainer
+	Tags                    ECSTaskTags
```

Comment:
It matches to 
https://github.com/DataDog/datadog-agent/blob/7d3819e712f3743a7fc3617c6ff3c9a8be56fe1a/pkg/util/ecs/metadata/v3or4/types.go#L16-L28